### PR TITLE
Added controls to the navigation view to use the policy ViewAny

### PR DESCRIPTION
**Pull request by erhansogut**

### CODE REVIEW

File: resources/views/navigation.blade.php [hunk 1]
```diff
@@ -1,32 +1,41 @@
-<h3 class="flex items-center font-normal text-white mb-6 text-base no-underline">
-    <svg class="sidebar-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-        <path fill="#B3C1D1"
-              d="M7 10V7a5 5 0 1 1 10 0v3h2a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-8c0-1.1.9-2 2-2h2zm2 0h6V7a3 3 0 0 0-6 0v3zm-4 2v8h14v-8H5zm7 2a1 1 0 0 1 1 1v2a1 1 0 0 1-2 0v-2a1 1 0 0 1 1-1z"/>
-    </svg>
-    <span class="sidebar-label">
-        Roles & Permissions
-    </span>
-</h3>
+@if (Auth::user()->can('viewAny', \Spatie\Permission\Models\Role::class) || Auth::user()->can('viewAny', \Spatie\Permission\Models\Permission::class))
```
Comment (vyuldashev) [inline resources/views/navigation.blade.php]: I guess it better to use class name from `config/permission.php`.

You can use `app(PermissionRegistrar::class)->getRoleClass()` for Role and `app(PermissionRegistrar::class)->getPermissionClass()` for Permission.

Comment (vyuldashev) [reply resources/views/navigation.blade.php]: Same goes to below.

Comment (erhansogut) [reply resources/views/navigation.blade.php]: Done

**Comment by vyuldashev** [comment]
Thanks. Tagged v1.1.4

**Pull request merged**